components/authz-service/engine/opa/policy/introspection_v2.rego

```diff
@@ -46,7 +46,8 @@ authorized_pair[pair] {
 
 allowed_project[project] {
 	project := policies[pol_id].statements[statement_id].projects[_]
-	match_pair[["allow", _, pol_id, statement_id]]
+	"allow" == policies[pol_id].statements[statement_id].effect
+	authz.has_member[pol_id]
 	not policies[pol_id].type == const_system_type
```

Comment:
Is this still necessary? If it's a system policy allowing something, a why exclude it? (but I hadn't been around when this was introduced)

Reply:
Indeed it is: system policies have wildcard projects, which means a normal user will get literally all projects as their allowed list. This line prevents that from happening.

Reply:
Hmm we could exclude policies with wildcard projects then, rather, couldn't we? 😃

Reply:
No, we do not want to do that if it is not a system policy. The above effect _should_ result in all  projects if the user has that access.

Reply:
Ah I see. Thanks!